runtime/src/chpl-topo.c

```diff
@@ -90,6 +90,23 @@ void chpl_topo_init(void) {
     return;
   }
 
+  // Check hwloc API version.
```

Comment:
The bugfix component of the version is available, just not very easily.  It's in the 'Version:' line in the `$CHPL_HOME/install/.../lib/pkgconfig/hwloc.pc` file.  If we really want it, `printchplenv --make` could produce it into a CHPL_MAKE setting and we could arrange to -D it into the runtime compilations so we could check it here.  But if we don't really need it then what is here is fine.